R/get_spawning_adults.R

```diff
@@ -67,7 +69,7 @@ get_spawning_adults <- function(year, adults, hatch_adults, mode,
     })
 
     init_adults <- rowSums(adults_by_month_hatchery_removed)
-    proportion_natural <- 1 - winterRunDSM::params$proportion_hatchery
+    proportion_natural <- 1 - winterRunDSM::params$proportion_hatchery #TODO should we make this into a param?
```

Comment:
Do we want to make `proportion_hatchery` and `natural_adult_removal_rate` into parameters for `get_spawning_adults()`